R/summaries.R

```diff
@@ -368,7 +368,13 @@ setMethod("countDominantFeatures", signature = c(x = "SummarizedExperiment"),
 #'
 #' @export
 setMethod("summary", signature = c(object = "SummarizedExperiment"),
-    function(object, assay_name = abund_values, abund_values = "counts"){
+    function(object, assay_name = NULL, abund_values = NULL){
+        # Specify assay names for user
+        message("Following assays detected: ", call. = FALSE)
+        print(assayNames(object))
+        # Check if assay name is specified
+        .check_abund_assay(object,
+                           assay_name = assay_name)
```

Comment:
Does `.check_assay_present` from utils.R work here?

Reply:
@microsud

Reply:
yes. resolving this.